tests/network/hp/procurve/snmp/spanning-tree.robot

```diff
@@ -0,0 +1,33 @@
+            ...      4     --critical-status='\\\%{op_status} ne "up"'                                                             CRITICAL: Port 'Anonymized 147' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '1'] - Port 'Anonymized 026' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '10'] - Port 'Anonymized 232' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '11'] - Port 'Anonymized 093' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '12'] - Port 'Anonymized 058' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '13'] - Port 'Anonymized 118' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '14'] - Port 'Anonymized 158' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '15'] - Port 'Anonymized 191' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '16'] - Port 'Anonymized 160' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '17'] - Port 'Anonymized 188' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '18'] - Port 'Anonymized 034' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '19'] - Port 'Anonymized 054' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '21'] - Port 'Anonymized 189' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '22'] - Port 'Anonymized 204' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '24'] - Port 'Anonymized 026' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '25'] - Port 'Anonymized 135' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '26'] - Port 'Anonymized 182' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '27'] - Port 'Anonymized 088' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '3'] - Port 'Anonymized 220' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '4'] - Port 'Anonymized 003' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '5'] - Port 'Anonymized 118' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '6'] - Port 'Anonymized 192' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '7'] - Port 'Anonymized 203' spanning tree state is 'blocking' [op status: 'down'] [admin status: 'up'] [index: '9']
```

Comment:
can you remove some of the port from the snmpwalk file so the output is more readable ?
If you let only 4 port in the snmpwalk file the test will still check the concatenation of multiples element, and the output should be more readable.